src/vendor/register-service-worker.min.js

```diff
@@ -0,0 +1,6 @@
+// Load and register pre-caching Service Worker
```

Comment:
It would appear that the linter didn't like my minimized version. Not sure that it matters as this will either be served out as .gz. or .br and it's only a couple of lines anyways.